python/doc/examples/meta_modeling/polynomial_chaos_metamodel/plot_chaos_build_distribution.py

```diff
@@ -52,11 +85,16 @@
 distribution.getCopula()
 
 # %%
-# The previous call to `BuildDistribution` is what is done internally by the following constructor of `FunctionalChaosAlgorithm`.
+# The previous call to `BuildDistribution` is what is done internally by the
+# following constructor of `FunctionalChaosAlgorithm`.
 
 # %%
 algo = ot.FunctionalChaosAlgorithm(inputSample, outputSample)
 
+# %%
+# The previous constructor is the main topic of the example
+# :doc:`Create a polynomial chaos metamodel from a data set </auto_meta_modeling/polynomial_chaos_metamodel/plot_functional_chaos>`.
+
 # %%
 # Reset default settings
 ot.ResourceMap.Reload()
```

Comment:
```suggestion
ot.ResourceMap.Reload() # reset default settings
```
This looks odd otherwise:

![image](https://github.com/openturns/openturns/assets/31989332/9087fe4a-28e5-489b-b198-99f9f2f4c3f9)